back-end/aws-cloud9-api/src/main/java/com/serverless/dal/QuestionDBTable.java

```diff
@@ -0,0 +1,199 @@
+        	if(results.get(i).getForm_membership().equals(formid))
+        	{
+        		res.add(results.get(i));
+        	}
+        	}
+        }
+        
+        
+        return res;
+    }
+    
+    public  Boolean deleteQuestionFromForm(String questionid) throws IOException {
+    	Question question = null;
+
+        HashMap<String, AttributeValue> av = new HashMap<String, AttributeValue>();
+        av.put(":v1", new AttributeValue().withS(questionid));
+
+        DynamoDBQueryExpression<Question> queryExp = new DynamoDBQueryExpression<Question>()
+            .withKeyConditionExpression("id = :v1")
+            .withExpressionAttributeValues(av);
+
+        PaginatedQueryList<Question> result = this.mapper.query(Question.class, queryExp);
+        if (result.size() > 0) {
+        	question = result.get(0);
+
+        }
+        
+        question.setForm_membership(null);
+        this.mapper.save(question);
+        
+        return true;
+    }
+    
+    public  Boolean createMembership(String questionid, String formid) throws IOException {
+    	Question question = null;
+
+        HashMap<String, AttributeValue> av = new HashMap<String, AttributeValue>();
+        av.put(":v1", new AttributeValue().withS(questionid));
+
+        DynamoDBQueryExpression<Question> queryExp = new DynamoDBQueryExpression<Question>()
+            .withKeyConditionExpression("id = :v1")
+            .withExpressionAttributeValues(av);
+
+        PaginatedQueryList<Question> result = this.mapper.query(Question.class, queryExp);
+        if (result.size() > 0) {
+        	question = result.get(0);
+
+        }
+        
+        question.setForm_membership(formid);
+        this.mapper.save(question);
+        
+        return true;
+    }
+    
+    
+    public  ClosedQuestion setCorrectAnswerr(String id, String answer) throws IOException {
+    	ClosedQuestion closed = null;
+
+        HashMap<String, AttributeValue> av = new HashMap<String, AttributeValue>();
+        av.put(":v1", new AttributeValue().withS(id));
+
+        DynamoDBQueryExpression<ClosedQuestion> queryExp = new DynamoDBQueryExpression<ClosedQuestion>()
+            .withKeyConditionExpression("id = :v1")
+            .withExpressionAttributeValues(av);
+
+        PaginatedQueryList<ClosedQuestion> result = this.mapper.query(ClosedQuestion.class, queryExp);
+        if(result.size() > 0) {
+        	closed = result.get(0);
+        }
+        closed.setCorrect_answer(answer);
+       return closed;
+    }
+    
+    
+    
+    
+    
+    public void save(Question newquestion) throws IOException {
+        this.mapper.save(newquestion);
+    }
+    
+    public void save(ClosedQuestion newquestion) throws IOException {
+        this.mapper.save(newquestion);
+    }
+    
+
+    
+ 
+    
+    public Boolean delete(String id) throws IOException {
+    	Question question = null;
+
+        // get product if exists
+    	question = get(id);
```

Comment:
1. Można w jednej linijce.
2. Jak używasz metod z tej klasy, to proszę o this
`question = this.get(id);`